TestProjects/UniversalGraphicsTest_Lighting/Assets/Test/Editor/CopyImageToReferenceFolders.cs

```diff
@@ -0,0 +1,79 @@
+using UnityEngine;
+using UnityEditor;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+public class CopyImageToReferenceFolders
```

Comment:
Why duplicate the file? It is already available in com.unity.testing.urp/Scripts/Editor/CopyImageToReferenceFolders.cs